app/rss/routes.py

```diff
@@ -0,0 +1,97 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+import json
+from datetime import datetime
+
+from app.api.schemas.components import ComponentSchema
+from app.models import Component
+from app.rss import bp
+
+from feedgen.feed import FeedGenerator
+
+from flask import Response
+
+
+@bp.route("/rss/<string:region>/<string:component_name>")
+def rss(region, component_name):
+    attr_name = "region"
+    attr_value = region
+    attribute = {attr_name: attr_value}
+    component = Component.find_by_name_and_attributes(
+        component_name, attribute
+    )
+    if not component:
+        return "Component not found", 404
+    component_data = ComponentSchema().dump(component)
+    incidents = component_data["incidents"]
+    none_end_date_list = []
+    sorted_list = []
+    limited_incidents = []
+    for incident in incidents:
+        if incident["end_date"] is None:
```

Comment:
I do not get the idea here. Why do you do this?

Reply:
First thought was because I wanted to add an always active Incident to the output to show if it was changed manually through the request to the database which is contrary to the design of the app. It's not helpful maybe but I thought it would be helpful